### DISCUSSION TITLE
能否调整中文扉页姓名~学院这几行中字间距？

### DISCUSSION THREAD

**Discussion by hotpot-fish** [sjtug/SJTUThesis#960] [提问]
又被图书管理员提意见了，希望扉页中前几行的缩小字间距，请问有什么方法吗？
![jwc](https://github.com/sjtug/SJTUThesis/assets/74254099/c96843ad-fd00-42ca-8189-718ba6ff3d6e)
![12](https://github.com/sjtug/SJTUThesis/assets/74254099/0b5367bd-c07c-47b7-adf4-f9aaf0632631)

**Comment by LogCreative** [comment]
> 如无必要，无需作出这种修改。

https://github.com/sjtug/SJTUThesis/blob/8e403d73e1f2402e07c21db784d264c5b92b8753/texmf/tex/latex/sjtutex/sjtu-name-thesis-zh.def#L45-L49

以第一行为例，改为
```latex
 { author           } { \hfill 姓\quad 名                   } , 
```
